src/random.cpp

```diff
@@ -39,7 +39,7 @@
 #include <openssl/err.h>
 #include <openssl/rand.h>
 
-static void RandFailure()
+[[noreturn]] static void RandFailure()
 {
     LogPrintf("Failed to read randomness, aborting\n");
     abort();
```

Comment:
Maybe change this to std::abort() so that its clearer that we're really [[noreturn]]ing?